src/sss_client/pam_sss.c

```diff
@@ -1913,7 +1913,7 @@ static int prompt_passkey(pam_handle_t *pamh, struct pam_items *pi,
      * the pointers to the struct pam_message are arrays. Since the assignment
      * makes clear that mesg[] and (*mesg)[] are arrays it should be kept this
      * way and not be replaced by other equivalent assignments. */
-    for (int i = 1; i < msg_idx; i++) {
+    for (int i = 0; i < msg_idx; i++) {
```

Comment:
Hi,

I think you do not have to change this, `mesg[0]` is already set in line 1908.

bye,
Sumit

Reply:
Thanks, I missed that. Fixed.